packages/patternfly-4/react-core/src/components/Pagination/OptionsToggle.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/OptionsMenu/options-menu';
+import { css, getModifier } from '@patternfly/react-styles';
+
+import { fillTemplate } from '../../helpers';
+import { ToggleTemplateProps } from './ToggleTemplate';
+import { DropdownToggle } from '../Dropdown';
+
+export interface OptionsToggleProps extends React.HTMLProps<HTMLDivElement> {
+  /** The type or title of the items being paginated */
+  itemsTitle?: string,
+  /** The text to be displayed on the Options Toggle */
+  optionsToggle?: string,
+  /** The first index of the items being paginated */
+  firstIndex?: number,
+  /** The last index of the items being paginated */
+  lastIndex?: number,
+  /** The total number of items being paginated */
+  itemCount?: number,
+  /** Id added to the title of the Pagination Options Menu */
+  widgetId?: string,
+  /** showToggle */
+  showToggle?: boolean,
+  /** Event function that fires when user clicks the Options Menu toggle */
+  onToggle?: (isOpen: boolean) => void,
+  /** Flag indicating if the Options Menu dropdown is open or not */
+  isOpen?: boolean,
+  /** */
+  parentRef?: HTMLElement;
+  /** This will be shown in pagination toggle span. You can use firstIndex, lastIndex, itemCount, itemsTitle props. */
+  toggleTemplate?: ((props: ToggleTemplateProps) => React.ReactElement) | string;
```

Comment:
This is much better than using `React.ReactNode` and then casting to `any`. Great job!